app/assets/stylesheets/about.scss

```diff
@@ -0,0 +1,116 @@
+// Place all the styles related to the aboutme controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+
+.name 
+h1 {
+    font-size: 150px ;
+    text-align:center;
+    font-family:fantasy;
+    padding-top:50px;
+    position: relative;
+    margin: 0px 0px 0px 0px;
+    padding: 0px 150px 0px 150px;
```

Comment:
You can use shorthand to more quickly set these duplicated values.

For the two above you could shorten these to

```
margin: 0;
padding: 0 150px;

You can read more about CSS shorthand here: https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties
```

Reply:
I can't believe I didnt spot this, fixed now, thanks for the info too